### PR TITLE
BI-535-Support download artifacts

### PR DESCRIPTION
**Pull request by sonnguyen-logigear**

### CODE REVIEW

File: workspace/lib/lita/handlers/teamcity.rb [hunk 1]
```diff
@@ -130,6 +130,15 @@ class Teamcity < Handler
           t('help.queue.syntax_wild') => t('help.queue.desc_wild')
         }
       )
+      
+      route(
+        /.*artifacts(.*)/,
```
Comment (scopp) [inline workspace/lib/lita/handlers/teamcity.rb]: why is this regex different than the other routes?

Comment (sonnguyen-logigear) [reply workspace/lib/lita/handlers/teamcity.rb]: I just want one route for this function, when user don't pass build_number, we will get latest build (success and finished) to get artifacts.For the consistency routes, I will split this to two route like queue/running command

File: workspace/locales/en.yml [hunk 2]
```diff
@@ -37,6 +37,11 @@ en:
             desc: "Lists all queued builds"
             syntax_wild: "*@buildbot queue `some_text*`*"
             desc_wild: "Lists queued builds matching a case-sensitive wildcard search with `*`"
+          artifacts:
+            syntax: "*@buildbot artifacts <build_name>"
```
Comment (scopp) [inline workspace/locales/en.yml]: We need to be careful with this syntax.  Should be:
```
*@buildbot artifacts `build_name`*
```

File: workspace/locales/en.yml [hunk 3]
```diff
@@ -37,6 +37,11 @@ en:
...
+            desc: "Lists all artifacts of latest build"
+            syntax_specific: "*@buildbot artifacts <build_name> [build_number]"
```
Comment (scopp) [inline workspace/locales/en.yml]:
```
*@buildbot artifacts `build_name` `build_number`*
```

**Pull request merged**